catalog/core/catalog-core-migratable/src/main/java/org/codice/ddf/catalog/migratable/impl/CatalogMigratableConfig.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.migratable.impl;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.regex.Pattern;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CatalogMigratableConfig {
+
+    //  Pkg pvt bounds on the property values - to ensure relationships are preserved in test
+
+    static final int MAX_CARDS_PER_FILE = 50000;
+
+    static final int MAX_QUERY_PAGE_SIZE = 100000;
+
+    static final int MIN_QUERY_PAGE_SIZE = 1000;
+
+    static final int MAX_THREADS = 20;
```

Comment:
The reason is that we will eventually expose this on the admin UI so admins _could_ set this too high and just kill their system. Having said that, a good max today may not be a good one two years from now so I'd be ok with not enforcing an upper limit.